packages/edit-post/src/components/header/style.scss

```diff
@@ -57,7 +80,7 @@
 .edit-post-header__settings {
 	display: inline-flex;
 	align-items: center;
-	flex-wrap: wrap;
+	flex-wrap: nowrap;
```

Comment:
Necessary to force the block toolbar to shrink. Otherwise the settings panel will continue to wrap buttons (which looks broken anyways).